.github/workflows/pull-request.yml

```
@@ -91,538 +48,356 @@
+      - name: Set up Python
+        uses: actions/setup-python@v5
         with:
-          files: |
-            .github/**
-            .husky/**
-            scripts/**
-            sample_data/**
-            .env.sample
+          python-version: 3.9
+
+            #- name: Run Biome Ignore Check
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)